setup_configure.py

```diff
@@ -58,6 +58,7 @@ class EnvironmentOptions(object):
     def __init__(self):
         self.hdf5 = os.environ.get('HDF5_DIR')
         self.hdf5_version = os.environ.get('HDF5_VERSION')
+        self.mpi = os.environ.get('HDF5_PARALLEL') == "ON"
```

Comment:
I thought about this first, too.
Unfortunately one would then expect that =FALSE,0,OFF should work too and I was not sure where to end...

Just "setting" a var in bash to smth. is therefore no great option.